### ISSUE TITLE
Android nightly builds failing

### ISSUE THREAD

**Issue by jfirebaugh** [mapbox/mapbox-gl-native#8474]
https://www.bitrise.io/build/d1d77cf85a2b1a53

```
:MapboxGLAndroidSDKTestApp:lintVitalRelease/bitrise/src/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/BubbleLayout.java:38: Error: This class should be public (com.mapbox.mapboxsdk.annotations.BubbleLayout) [Instantiatable]
  public BubbleLayout(Context context, AttributeSet attrs, int defStyleAttr) {
         ~~~~~~~~~~~~

   Explanation for issues of type "Instantiatable":
   Activities, services, broadcast receivers etc. registered in the manifest
   file (or for custom views, in a layout file) must be "instantiatable" by
   the system, which means that the class must be public, it must have an
   empty public constructor, and if it's an inner class, it must be a static
   inner class.

1 errors, 0 warnings
```

**Comment by zugaldia** [comment]
Related: https://github.com/mapbox/mapbox-gl-native/pull/8356.

**Comment by jfirebaugh** [comment]
Should be fixed by #8479. Manually triggered build happening here: https://www.bitrise.io/build/9930526cb0f69d01
